### ISSUE TITLE
Port numbers not updated by install script

### ISSUE THREAD

**Issue by phaumer** [zowe/zowe-install-packaging#29]
I specified new port numbers for the explorer-server in zowe-install.yaml file and ran the install script. The script reports that it will use the ports I specified and does not report any errors, but after install finished I still see the default port number 7443 in these files:

- /explorer-JES/web/index.html
- /explorer-MVS/web/index.html
- /explorer-USS/web/index.html

I have to manually edit them to make the three browsers work inside the MVD.

**Comment by jackjia-ibm** [comment]
Tested with latest build #28, zowe-0.9.0-20180914.151934-28.pax, this issue doesn't exist any more. So should have been resolved by some other fixes.

**Comment by jackjia-ibm** [comment]
Tested with released build v0.9.0, cannot reproduce either. So this could be related to the permission settings which doesn't happen on zD&T.

**Comment by jackjia-ibm** [comment]
@phaumer we have v0.9.1 now, could you please verify if this issue still exists? Thank you!

**Comment by bjsimmons606** [comment]
@jackjia-ibm I just went through the 0.9.1 install and came across a similar issue.  The script did change the ports to what was assigned but I believe the explorer iframes within the mvd are calling the http port instead of the https port that I assigned resulting in a "Secure Connection Failed" error.  I fixed the issue by changing the ports in the explorer's index.html files but there may be an issue in the script assigning the correct port to the explorer iframe calls or the explorer iframes are asking for the http port instead of the https.
